src/content/vintage/use-the-api/management-api/cluster-apps/default-apps-aws.md

```diff
@@ -38,9 +38,9 @@ This page lists all available configuration options, based on the [configuration
   <div class="property depth-0">
     <div class="property-header">
       <h3 class="property-path headline-with-link">
-        <a class="header-link" href="#apps-aws-pod-identity-webhook">
+        <a class="header-link" href="#userConfig-certManager">
           <i class="fa fa-link"></i>
-        </a>.apps.aws-pod-identity-webhook</h3>
+        </a>.userConfig.certManager</h3>
```

Comment:
@pipo02mix Is the generated HTML maybe not idempotent (e.g. requires sorting of the `.Values.xyz` fields)?

Reply:
I need this PR merged, I will go on an merge it despite this problem, sorry